src/lib/prepareOas.ts

```diff
@@ -65,9 +65,18 @@ export default async function prepareOas(path: string, command: 'openapi' | 'ope
           const oas = new OASNormalize(file, { enablePaths: true });
           return oas
             .version()
```

Comment:
I ended up bringing back `.version() in `oas-normalize@8.1.0` (https://github.com/readmeio/oas-normalize/pull/227) but slightly changed the original API it had to give us some more information on the file it's been supplied.